### PR TITLE
[12.0][MIG] mail_drop_target

### PR DESCRIPTION
**Pull request by etobella**
Standard migration.

Applied changes:
* Change of version
* Update of oca-gen-adon-readme

@hbrunn @jarroyomorales @JordiBForgeFlow

### CODE REVIEW

**Comment by etobella** [comment]
It was needed to add a context verification value on _notify due to https://github.com/odoo/odoo/commit/d139e92c621f88676e5fefa9f2359dabc8b4602c#diff-86c5f1ed15bfa23d6be2285dc88e8a57

File: mail_drop_target/models/mail_thread.py [hunk 1]
```diff
@@ -18,6 +18,8 @@ def message_drop(self, model, message, custom_values=None,
         disable_notify_mail_drop_target = \
             self.env["ir.config_parameter"].sudo().get_param(
                 "mail_drop_target.disable_notify", default=False)
+        import logging
+        logging.info(disable_notify_mail_drop_target)
```
Comment (jarroyomorales) [inline mail_drop_target/models/mail_thread.py]: This isnt necessary, is it?

Comment (etobella) [reply mail_drop_target/models/mail_thread.py]: Fixed

**Comment by LoisRForgeFlow** [comment]
@etobella There are some conflics, can you check?

**Comment by etobella** [comment]
Fixed!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

**Review by jarroyomorales — APPROVED** [review]
Functional test and works as expected!

**Pull request merged**